compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/diagnostics/DiagnosticErrorCode.java

```diff
@@ -318,7 +318,8 @@
     ERROR_DEFAULTABLE_PARAMETER_CANNOT_BE_INCLUDED_RECORD_PARAMETER("BCE664",
             "error.defaultable.parameter.cannot.be.included.record.parameter"),
     ERROR_INCOMPLETE_QUOTED_IDENTIFIER("BCE665", "error.incomplete.quoted.identifier"),
-    ERROR_OPEN_RECORD_CANNOT_CONTAIN_REST_FIELD("BCE666", "error.open.record.cannot.contain.rest.field"),
+    ERROR_INCLUSIVE_RECORD_TYPE_CANNOT_CONTAIN_REST_FIELD("BCE666",
+            "error.inclusive.record.type.descriptor.cannot.contain.rest.field"),
```

Comment:
We usually use the same dotted version of the name right?
```suggestion
    ERROR_INCLUSIVE_RECORD_TYPE_CANNOT_CONTAIN_REST_FIELD("BCE666",
            "error.inclusive.record.type.cannot.contain.rest.field"),
```